schema/crdb/dbinit.sql

```diff
@@ -4214,7 +4243,7 @@ INSERT INTO omicron.public.db_metadata (
     version,
     target_version
 ) VALUES
-    (TRUE, NOW(), NOW(), '86.0.0', NULL)
+    (TRUE, NOW(), NOW(), '87.0.0', NULL)
```

Comment:
This is in conflict with main now.  I will merge main and bump this when we are closing to landing this PR

Reply:
up-to-date again because I wanted to make sure the sled-agent dropshot commit didn't cause conflicts.